CoreSchemaView/src/au/gov/asd/tac/constellation/views/schemaview/TypeDropper.java

```diff
@@ -164,4 +103,89 @@ public void edit(final GraphWriteMethods wg, final PluginInteraction interaction
         }
         return null;
     }
+
+    /**
+     * Plugin to help once the type had been dropped on a vertex
+     */
+    @PluginInfo(pluginType = PluginType.UPDATE, tags = {"MODIFY"})
+    public static class TypeDropperPlugin extends SimpleEditPlugin {
```

Comment:
I was thinking "IMPORT" because what the `*Dropper`'s allow you to do is drag and drop something onto the graph and it should render it.